stubs/Collection.stub

```diff
@@ -38,9 +41,10 @@ class Collection implements \ArrayAccess, Enumerable
     public function pop() {}
 
     /**
-     * @param  mixed  $key
-     * @param  mixed  $default
-     * @return TValue|null
+     * @template TDefault
+     * @param    mixed  $key
```

Comment:
I think this can be just `string` or `int` isn't it? In that case the type here can be `array-key` instead of `mixed`